assets/javascripts/modules/analytics/setup.js

```diff
@@ -44,7 +44,10 @@ define([
          * GA and other tags often prefer Ophan to have bootstrapped and set window.guardian state,
          * so load them after Ophan has loaded if thirdPartyTrackingEnabled.
          */
-        ophan.loaded.finally(() => cmp.registerCallbackOnConsentChange(loadGA));
+        ophan.loaded.finally(() => {
+            cmp.createPrivacySettingsLink();
```

Comment:
I think this `cmp.createPrivacySettingsLink();` would be better placed in the `main.js` file, after we've initiliased the cmp: https://github.com/guardian/subscriptions-frontend/blob/f85be04da5f3bd60eb5a97683e173d8ad80747ee/assets/javascripts/main.js#L56

Reply:
👍 Both changes made, thanks!